src/main/java/spoon/support/compiler/jdt/JDTTreeBuilder.java

```diff
@@ -1358,9 +1358,11 @@ public boolean visit(LabeledStatement labeledStatement, BlockScope scope) {
 
 	@Override
 	public boolean visit(LocalDeclaration localDeclaration, BlockScope scope) {
+		// an unnamed pattern does not have a type in the source, but the binding holds the inferred type
+		CtElement element;
```

Comment:
Why is this declared up here if it is only used and assigned once, much further down?

Reply:
It was a leftover from a different approach that failed miserably.